utils/scripts/install-centos8.sh

```diff
@@ -64,6 +64,7 @@ dnf --nodocs install \
     python3-tabulate \
     python3-yaml \
     sg3_utils \
+    sudo \
```

Comment:
Why/when is `sudo` used?  Haven't we had a history of bad things happening when `sudo` is used in containers?  It tends to end up with files owned by root in the workspace that cannot be cleaned up by Jenkins.

Reply:
Actually the opposite, I tried to remove the use of sudo in https://daosio.atlassian.net/browse/DAOS-9687 however doing so caused the problems with the files owned by root.  The use of sudo is https://github.com/daos-stack/daos/blob/master/utils/docker_nlt.sh#L13-L32 and remains uncahnged.  We could in theory remove the sudo package/setup from the centos 7 dockerfile as part of this PR I believe.